tasks/main.yml

```diff
@@ -12,6 +12,13 @@
     state: present
   when: keepalived_install
 
+- name: Ensure default keepalived conf is installed
+  template:
+    src: keepalived.default
+    dest: /etc/default/keepalived
+    mode: 0644
+  notify: Restart keepalived
+
```

Comment:
This is probably Debian only but as I don't have any other installation target this is fine for now.

Reply:
Indeed, didn't find same file on redhat for example.